src/commands/sentinel-slaves.json

```
@@ -0,0 +1,27 @@
+{
+    "SLAVES": {
+        "summary": "List the monitored slaves",
+        "complexity": "O(N) where N is the number of slaves",
+        "group": "sentinel",
+        "since": "3.0.0",
```

isn't it supported since 2.8?
```suggestion
        "since": "2.8.0",
```

---

Looks like it: https://github.com/redis/redis/blob/2.8/src/sentinel.c#L2656